lib/ApiRequestor.php

```diff
@@ -121,9 +121,9 @@ private function _requestRaw($method, $url, $params, $headers)
 
         if (!$myApiKey) {
             $msg = 'No API key provided.  (HINT: set your API key using '
-              . '"Stripe::setApiKey(<API-KEY>)".  You can generate API keys from '
-              . 'the Stripe web interface.  See https://stripe.com/api for '
-              . 'details, or email support@stripe.com if you have any questions.';
+              . '"Stripe::setApiKey(<API-KEY>)".  You can find your API keys '
```

Comment:
Don't a number of api returned messages return mentions of support and the dashboard? Like `unable to use live transfers, please email support` or `managed accounts not enabled please register your platform`? It seems like this is poor error handling on behalf of the person integrating Stripe